README.md

```diff
@@ -97,7 +97,7 @@ and uninstalled through
 
     $ sudo make uninstall.comp
 
-`DESTDIR` is supported. Please refer to your shell's manual for instructions on completion configuration.
+Again, `PREFIX` is supported. Please refer to your shell's manual for instructions on completion configuration.
```

Comment:
Adding `PREFIX=/usr` in the example would render this line redundant. And we an also skip the `read your shell's manual` part.